package.json

```diff
@@ -328,92 +328,92 @@
 		"keytar@npm:7.7.0/node-addon-api": "3.1.0",
 		"lzma-native": "^8.0.6",
 		"node-abi": "3.35.0",
-		"@wordpress/a11y": "4.8.0",
-		"@wordpress/annotations": "3.8.0",
-		"@wordpress/api-fetch": "7.8.0",
-		"@wordpress/autop": "4.8.0",
-		"@wordpress/babel-plugin-import-jsx-pragma": "5.8.0",
-		"@wordpress/babel-plugin-makepot": "6.8.0",
-		"@wordpress/babel-preset-default": "8.8.2",
-		"@wordpress/base-styles": "^5.8.0",
-		"@wordpress/blob": "4.8.0",
-		"@wordpress/block-directory": "5.8.3",
-		"@wordpress/block-editor": "^14.3.0",
-		"@wordpress/block-library": "^9.8.2",
-		"@wordpress/block-serialization-default-parser": "5.8.0",
-		"@wordpress/block-serialization-spec-parser": "5.8.0",
-		"@wordpress/blocks": "^13.8.0",
-		"@wordpress/browserslist-config": "6.8.0",
-		"@wordpress/components": "^28.8.0",
-		"@wordpress/compose": "7.8.0",
-		"@wordpress/core-data": "7.8.0",
-		"@wordpress/create-block-tutorial-template": "4.8.0",
-		"@wordpress/create-block": "4.51.0",
+		"@wordpress/a11y": "4.14.0",
+		"@wordpress/annotations": "3.14.0",
+		"@wordpress/api-fetch": "7.14.0",
+		"@wordpress/autop": "4.14.0",
+		"@wordpress/babel-plugin-import-jsx-pragma": "5.14.0",
+		"@wordpress/babel-plugin-makepot": "6.14.0",
+		"@wordpress/babel-preset-default": "8.14.0",
+		"@wordpress/base-styles": "^5.14.0",
+		"@wordpress/blob": "4.14.0",
+		"@wordpress/block-directory": "5.14.0",
+		"@wordpress/block-editor": "^14.9.0",
+		"@wordpress/block-library": "^9.14.0",
+		"@wordpress/block-serialization-default-parser": "5.14.0",
+		"@wordpress/block-serialization-spec-parser": "5.14.0",
+		"@wordpress/blocks": "^14.3.0",
+		"@wordpress/browserslist-config": "6.14.0",
+		"@wordpress/components": "^29.0.0",
+		"@wordpress/compose": "7.14.0",
+		"@wordpress/core-data": "7.14.0",
+		"@wordpress/create-block-tutorial-template": "4.14.0",
+		"@wordpress/create-block": "4.57.0",
 		"@wordpress/custom-templated-path-webpack-plugin": "2.13.0",
-		"@wordpress/customize-widgets": "5.8.2",
-		"@wordpress/data-controls": "4.8.0",
-		"@wordpress/data": "^10.8.0",
-		"@wordpress/dataviews": "4.4.0",
-		"@wordpress/date": "5.8.0",
+		"@wordpress/customize-widgets": "5.14.0",
+		"@wordpress/data-controls": "4.14.0",
+		"@wordpress/data": "^10.14.0",
+		"@wordpress/dataviews": "4.10.0",
```

Comment:
Got this when trying to start Calypso:

```
Error: No "exports" main defined in /Users/fushar/Code/Automattic/wp-calypso/node_modules/@wordpress/dataviews/package.json
```

It seems there's an open issue related to this: https://github.com/WordPress/gutenberg/issues/67897

I see there is a potential fix, but there hasn't been any new version for this package.